src/relay.ts

```diff
@@ -46,25 +44,30 @@ export const relay = async (request: any, response: any): Promise<void> =>  {
 
   if (!chainConfigInfo) return;
 
-  const relayInfo = JSON.stringify({ chainId, signedVAA });
-  console.log(`relaying: ${relayInfo}`);
+  const relayInfo = { chainId, signedVAA };
+  logger.debug(relayInfo, 'relaying ...');
 
   try {
     const receipt = await relayEVM(chainConfigInfo, signedVAA);
+    logger.debug({ ...relayInfo, txHash: receipt.transactionHash }, 'Relay Succeed 🎉🎉');
 
-    console.log(`Relay Succeed 🎉🎉: ${relayInfo}, txHash: ${receipt.transactionHash}`);
-    response.status(200).json(receipt);
+    return response.status(200).json(receipt);
   } catch (e) {
-    console.log(`Relay Failed ❌: ${relayInfo}`);
-    console.error(e);
-    return response.status(500).json({ error: e, msg: 'Unable to relay this request.' });
+    logger.debug(relayInfo, 'Relay Failed ❌');
+    logger.error(e);
+
+    return response.status(200).json({
```

Comment:
use 200 to be consistent with router endpoints. current 500 actually broke womrhole prod UI when there is relay error